docroot/noc/index.html

```diff
@@ -27,7 +27,7 @@ <h2 style="border-bottom: solid 1px silver;">Wikimedia NOC</h2>
 		   <dd>Core DB Replication Layout and Lag</dd>
 		   <dt><a href="./db.php">DB clusters</a></dt>
 		   <dd>Database cluster info</dd>
-		<dt><a href="https://ganglia.wikimedia.org/">Ganglia</a></dt>
+		<dt><a href="https://grafana.wikimedia.org/">Grafana</a> (new) or <a href="https://ganglia.wikimedia.org/">Ganglia</a> (old)</dt>
```

Comment:
Entirely up to you! (I just noticed the outdated link, and put up a best-guess for the replacement. I don't use either of them, so am not the target demographic...)